distribution/istio-resources/kustomization.yaml

```diff
@@ -2,11 +2,14 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- deny-all-authorizationpolicy.yaml
 - envoy-filter-ingressgateway-settings.yaml
 - envoy-filter-kubeflow-userid.yaml
 - envoy-filter-proxy-protocol.yaml
 - gateway-authorizationpolicy.yaml
+- ingress-class.yaml
 - kubeflow-cluster-roles.yaml
 - kubeflow-gateway.yaml
-# - monitoring/
+- monitoring/
```

Comment:
Probably shouldn't have this enabled by default since it will cause deployments to fail if the kube-prometheus-stack isn't also deployed.